org.jitsi.jitsi-meet.yaml

```diff
@@ -0,0 +1,56 @@
+---
+app-id: org.jitsi.jitsi-meet
+base: org.electronjs.Electron2.BaseApp
+base-version: '19.08'
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: jitsi-meet-run
+separate-locales: false
+tags: []
+finish-args:
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: unappimage
+    buildsystem: simple
+    build-commands:
+      - make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=/app/bin
+    sources:
+      - type: git
+        url: https://github.com/refi64/unappimage
+        commit: d7f86f2a0d7ec3a69211125207d5f127386b849a
+
+  - name: jitsi-meet
+    buildsystem: simple
+    build-commands:
+      - unappimage jitsi-meet-x86_64.AppImage
+      - rm jitsi-meet-x86_64.AppImage
+
+      - sed -e 's/Exec=.*/Exec=jitsi-meet-run/'
```

Comment:
You can use desktop-file-edit here btw

Reply:
Don't think `desktop-file-edit` can change the `Exec` key though?

Reply:
It can change the value of any key I think

Reply:
It can change the value of any extended key (`X-*`), but for default keys there are only setters for Name, GenericName, Comment, and Icon. (As well as list adders/removers for categories, mime-types, and only show in environments)

So no, it can't change `Exec` - at least the version that's available in org.freedesktop.Sdk.

Reply:
`desktop-file-edit --set-key=Exec --set-value=something $desktop-file` works just fine here

Reply:
Yeah, I'm not able to recreate the exact error anymore here either, but the application shouted at me when I tried before. Maybe I managed to get an input error?